zetaclient/chains/solana/signer/signer.go

```diff
@@ -209,6 +211,56 @@ func (signer *Signer) TryProcessOutbound(
 	// set relayer balance metrics
 	signer.SetRelayerBalanceMetrics(ctx)
 
+	for range 30 {
```

Comment:
Could we maybe consider a sub function? This function starts getting very complex